src/plugins/intel_cpu/tests/unit/shape_inference_test/scatter_update_shape_inference_test.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <openvino/op/ops.hpp>
+#include <openvino/op/parameter.hpp>
+#include <utils/shape_inference/shape_inference.hpp>
+#include <utils/shape_inference/static_shape.hpp>
+
+using namespace ov;
+using namespace ov::intel_cpu;
+
+TEST(StaticShapeInferenceTest, ScatterUpdate_3D_axis_1) {
```

Comment:
Optional:
Consider usage of `OpStaticShapeInferenceTest` from test utils it has got defined some input/output shapes vectors and operator maker helper.
Can be use as parametrized test fixture or standard and some common part can be moved to fixture.

Reply:
Thank you for this suggestion. It can be used, but just for those few test cases, I don't see such need.